### PR TITLE
Caeser cipher

### PR DESCRIPTION
**Pull request by parjanyaacoder**
# Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Ceaser Cipher in Java

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#422

### CODE REVIEW

**Comment by parjanyaacoder** [comment]
@HarshCasper @Abhijit2505 Please review

**Review by mar-galho — APPROVED** [review]
The code works. I wouldn't mix nextLine and nextInt though; I'd rather receive the int value as a string and parse it - Scanners can get messy when you mix receiving input from texts and numbers in the same instance...

**Comment by parjanyaacoder** [comment]
@atarax665 @iamrajiv Please review.

File: Java/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ _add list here_
 _add list here_
 
 ## Cryptography
-_add list here_
+* [Ceaser Cipher](cryptography/ceaserCipher.java)
```
Comment (HarshCasper) [inline Java/README.md]: Correct the Spelling

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make the apt changes required and correct the spell mistakes. Ping me on Slack once you have made these changes.

**Pull request merged**